### ISSUE TITLE
Version conflict with httplib2

### ISSUE THREAD

**Issue by HackerMD** [gc3-uzh-ch/elasticluster#319]
After developer installation of the master branch:

```
git clone https://github.com/gc3-uzh-ch/elasticluster.git
cd ~/elasticluster
pip install -e . --user
```

I got the following error:

```
$ .local/bin/elasticluster
Traceback (most recent call last):
  File ".local/bin/elasticluster", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2991, in <module>
    @_call_aside
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2977, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3004, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 664, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 677, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 861, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (httplib2 0.8 (/usr/lib/python2.7/dist-packages), Requirement.parse('httplib2>=0.9.1'), set(['oauth2client']))
```

Upgrading the ``httplib2`` package solved the problem:

```
pip install -U httplib2 --user
```

**Comment by riccardomurri** [comment]
What base OS?  what exact version of Python?

**Comment by HackerMD** [comment]
Ubuntu 14.04
Python 2.7.12

**Comment by riccardomurri** [comment]
Fixed in commit 077e3de8320da4fc794061c7d3269f56061e8e69
